sdcm/nemesis.py

```diff
@@ -3161,6 +3175,10 @@ def disrupt_network_reject_thrift(self):
             wait_time=wait_time
         )
 
+    def _install_iptables(self) -> None:
+        if self.target_node.distro.is_ubuntu:  # iptables is missing in a minimized Ubuntu installation
+            self.target_node.remoter.sudo("apt install iptables")
```

Comment:
What happens when it’s already installed?
Do we have “fake errors” in the log?

Reply:
it would say something like:
```
iptables is already the newest version (1.8.7-1ubuntu5).
```

and returns without failure. (status code 0)

Reply:
Ok, that’s good enough